src/test/javascript/spec/component/programming-exercise/programming-exercise-configure-grading.component.spec.ts

```diff
@@ -775,24 +790,24 @@ describe('ProgrammingExerciseConfigureGradingComponent', () => {
 
         fixture.detectChanges();
 
-        expect(loadStatisticsStub).to.have.been.calledWithExactly(exerciseId);
+        expect(loadStatisticsStub).toHaveBeenCalledWith(exerciseId);
```

Comment:
Could you specify the number of calls here?

Reply:
:+1: It somehow gets called twice, I’m not sure why. Before removing an additional call in the [first of the new commits](https://github.com/ls1intum/Artemis/pull/4109/commits/6604629cf99e3136cd19697221389c36a11681c8#diff-59d509349f896039f207607619142488e8bba5cf462e1a234c5b6debf504d71fL181) it was called thrice.

On our prod system the page currently requests the statistics twice on page load (probably once directly and once via the `subscribeForTestCaseUpdates`). I assume that at least this should be fixed by removing the additional call. I will have to check on a test system that the statistics still work, and otherwise add the additional call back in.